static/schemas/config.evm.json

```diff
@@ -302,6 +288,7 @@
           "minimum": 0
         },
         "fallback_stall_timeout": {
+          "description": "If a fallback RPC is provided, the amount of time in ms to wait before kicking off the next provider",
```

Comment:
Maybe worse updating the `(default: false)` in the description for rollbacks, so it's included in the patch

Reply:
I think we need to implement serde defaults then...  But yes I agree will see what I can do.